microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -247,30 +248,24 @@ To use the Fixed Version distribution mode:
 
         *   Alternatively, use `EnsureCoreWebView2Async` ([WPF](/dotnet/api/microsoft.web.webview2.wpf.webview2.ensurecorewebview2async)/[WinForms](/dotnet/api/microsoft.web.webview2.winforms.webview2.ensurecorewebview2async)) to specify the environment.  Use the `browserExecutableFolder` parameter in [CoreWebView2Environment.CreateAsync](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createasync) to indicate the path to the Fixed Version binaries.
 
-1.  Package and ship the Fixed Version binaries with your app.  Update the binaries as appropriate.
-
-### Fixed Versioning with WinUI/UWP apps
-
-1. Download the corresponding fixed version binaries from the [WebView2 download](https://developer.microsoft.com/en-us/microsoft-edge/webview2/) site.
-    :::image type="content" source="../media/fixedversioning.png" alt-text="WebView2 Fixed Version." lightbox="../media/fixedversioning.png":::
-1. Extract the file contents (same as above steps), and copy to a directory under the project. Ex. ProjectName\95.0.1020.53
-1. Open the project file (.csproj) in a code editor and add the following code snippet within the project tags:
-    1. ```xml
-        <Content Include="FixedRuntime\95.0.1020.53\\**\*.*">
+    * For WinUI this might include opening the project file (.csproj) in a code editor and add the following code snippet within the project tags:
+        ```xml
+        <Content Include="FixedRuntime\(version number)\\**\*.*">
         <CopyToOutputDirectory>PreserveNewest<CopyToOutputDirector>
         </Content>
-    ```
-1. Rebuild your project and verify that the `bin\ **designated architecture**\Release` folder has a matching FixedRuntime\95.0.1020.53 folder with the runtime files in it
-1. Ensure the app has access to the folder by specifying the installed location and setting the environment variable for runtime path.
-    1. One way to do so is by adding the following c charp code to your app. 
+        ```
+    2. Verify that the `bin\ **designated architecture**\Release` folder has a matching FixedRuntime\95.0.1020.53 folder with the runtime files in it
```

Comment:
The number bullet here is showing up weird in the rendered page. Not sure what needs to change there.

There is an unnecessary space in 'bin\ ** designated...' that can be removed.

The version given in "FixedRuntime\..." should be "(version number)" instead of 95.0.1020.53.